updateVersion.sh

```diff
@@ -105,6 +105,7 @@ function merge_files() {
   AUTH_SRC=$SRC/istio-auth
   ISTIO=$ROOT/install/kubernetes/istio.yaml
   ISTIO_AUTH=$ROOT/install/kubernetes/istio-auth.yaml
+  ISTIO_AUTH_PER_CLUSTER_CA=$ROOT/install/kubernetes/templates/istio-auth/isio-auth-per-cluster-ca.yaml
```

Comment:
I agree "template" doesn't say the purpose. I was a little concerned about putting it in install/kubernetes because we may want to keep that directory clean. Note that per-cluster Istio CA yaml file is also in templates/istio-auth. I think maybe advanced users need to refer to tamplates/ to deploy components separately any way.